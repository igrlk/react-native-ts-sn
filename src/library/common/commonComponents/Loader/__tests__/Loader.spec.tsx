import React from 'react';
import renderer from 'react-test-renderer';

import Loader from '../Loader';

describe('Loader', () => {
	it('match to snapshot', () => {
		const tree = renderer.create(<Loader />).toJSON();
		expect(tree).toMatchSnapshot();
	});
});
