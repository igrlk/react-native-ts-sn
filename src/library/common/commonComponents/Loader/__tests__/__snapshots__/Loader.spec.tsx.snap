// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader match to snapshot 1`] = `
<ActivityIndicator
  animating={true}
  color="#999999"
  hidesWhenStopped={true}
  size="small"
/>
`;
